backend/btrixcloud/operator/crawls.py

```diff
@@ -325,8 +325,15 @@ async def _load_qa_configmap(self, params, children):
         qa_source_crawl_id = params["qa_source_crawl_id"]
         name = f"qa-replay-{qa_source_crawl_id}"
 
-        if name in children[CMAP]:
-            return [children[CMAP][name]]
+        configmap = children[CMAP].get(name)
+        if configmap and not self._qa_configmap_update_needed(name, configmap):
+            metadata = configmap["metadata"]
+            configmap["metadata"] = {
```

Comment:
need to copy these specifically to avoid re-adding extra metadata added by metacontroller